scripts/check_for_update.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+set -eu
+
+# get currently used Prisma CLI versions
+CURRENT_PRISMA_DEV_VERSION=$(cat scripts/versions/prisma_dev)
+CURRENT_PRISMA_LATEST_VERSION=$(cat scripts/versions/prisma_latest)
+CURRENT_PRISMA_PATCH_DEV_VERSION=$(cat scripts/versions/prisma_patch)
```

Comment:
File should probably also be called `prisma_patch_dev` or `prisma_patch-dev` for consistency.